code/provision/playbook.yml

```diff
@@ -58,3 +58,13 @@
 
     - name: Install jmespath
       pip: name=jmespath
+
+    - name: Install python3-pip
+      apt:
+        name: python3-pip
```

Comment:
I've got in my head that python3 causes Ansible hurt. I'm not saying that we shouldn't merge, just wanted to flag it so we both know if there are knock on effects later.

Reply:
Think that running Ansible using Python3 can be troublesome when mixing with Python2 servers. The provisioned control VM already has Python3 present without issue, this is just adding pip installer. Hopefully that won't cause hurt.